cime_config/SystemTests/sspmatrixcn.py

```diff
@@ -207,15 +207,18 @@ def run_phase(self):
                      linkfile = os.path.join(rundir, os.path.basename(item))
                      if os.path.exists(linkfile):
                          os.remove( linkfile )
-                     os.symlink(item, linkfile )
+                     self._case.flush()
+                     if os.path.exists(rundir):
+                         os.symlink(item, linkfile )
```

Comment:
The symlink line was triggering the error.
@ekluzek suggested moving up the "flush" line before the symlink line. Now the run directory gets created, though I'm getting a new error that the /timing directory is missing...

Reply:
Is this something that needs to be replicated in the run_neon code? if so, is it easily implemented in #2435?

Reply:
@wwieder and @TeaganKing it looks like it's not quite done the same way in run_neon. So you can't just easily move it over. However, tower_site.py in set_ref_case has calls to symlink_force, and I wonder if those are failing? And something similar needs to be done there?

Reply:
My most recent test of this change tells me that it did not fix anything.